packages/oauth/oauth_server.js

```diff
@@ -30,58 +30,57 @@ OAuth._requestHandlers = {};
 //       up in the user's services[name] field
 //     - `null` if the user declined to give permissions
 //
-OAuth.registerService = function (name, version, urls, handleOauthRequest) {
+OAuth.registerService = (name, version, urls, handleOauthRequest) => {
   if (registeredServices[name])
-    throw new Error("Already registered the " + name + " OAuth service");
+    throw new Error(`Already registered the ${name} OAuth service`);
 
   registeredServices[name] = {
     serviceName: name,
-    version: version,
-    urls: urls,
-    handleOauthRequest: handleOauthRequest
+    version,
+    urls,
+    handleOauthRequest,
   };
 };
 
 // For test cleanup.
-OAuthTest.unregisterService = function (name) {
+OAuthTest.unregisterService = name => {
   delete registeredServices[name];
 };
 
 
-OAuth.retrieveCredential = function(credentialToken, credentialSecret) {
-  return OAuth._retrievePendingCredential(credentialToken, credentialSecret);
-};
+OAuth.retrieveCredential = (credentialToken, credentialSecret) =>
+  OAuth._retrievePendingCredential(credentialToken, credentialSecret);
 
 
 // The state parameter is normally generated on the client using
 // `btoa`, but for tests we need a version that runs on the server.
 //
-OAuth._generateState = function (loginStyle, credentialToken, redirectUrl) {
+OAuth._generateState = (loginStyle, credentialToken, redirectUrl) => {
   return Buffer.from(JSON.stringify({
     loginStyle: loginStyle,
     credentialToken: credentialToken,
     redirectUrl: redirectUrl})).toString('base64');
 };
 
-OAuth._stateFromQuery = function (query) {
-  var string;
+OAuth._stateFromQuery = query => {
+  let string;
   try {
     string = Buffer.from(query.state, 'base64').toString('binary');
   } catch (e) {
-    Log.warn('Unable to base64 decode state from OAuth query: ' + query.state);
+    Log.warn(`Unable to base64 decode state from OAuth query: ${query.state}`);
     throw e;
   }
 
   try {
     return JSON.parse(string);
   } catch (e) {
-    Log.warn('Unable to parse state from OAuth query: ' + string);
+    Log.warn(`Unable to parse state from OAuth query: ${string}`);
     throw e;
   }
 };
 
-OAuth._loginStyleFromQuery = function (query) {
-  var style;
+OAuth._loginStyleFromQuery = quer => {
```

Comment:
Typo here. Missing `y`. Should be `query`.

Reply:
Wow, good 👀 @StorytellerCZ!